### ISSUE TITLE
`repo-add` does not create an empty database

### ISSUE THREAD

**Issue by rikushoney** [aurutils/aurutils#1168]
I recently did a fresh install of Arch and could no longer create an empty local repository following the steps in `man aur`.

After some digging I came across a change made in the upstream `repo-add` script ([f91fa546](https://gitlab.archlinux.org/pacman/pacman/-/commit/f91fa546f65af9ca7cdbe2b419c181df609969b7)) that causes it to not create a database if it is used without specifying any packages to add.

A quick workaround was simply adding the `aurutils` package (although it could work with any other AUR package) to the database after the package was built manually using `makepkg`, i.e.

```
sudo install -d /home/custompkgs -o $USER
repo-add /home/custompkgs/custom.db.tar.gz /path/to/aurutils.pkg.tar.zst
```

This has since been fixed in [6dddb1f7](https://gitlab.archlinux.org/pacman/pacman/-/commit/6dddb1f78cc1728d3842bc9b5e12ed9b0ff6a2a5?merge_request_iid=183) but hasn't been merged yet.

I thought mentioning the issue here will help others until the fix is merged or maybe there is an alternative solution that does not require adding any prebuilt packages to create the database.

**Comment by AladW** [comment]
You can run
```
tar zcf custom.db.tar.gz -T /dev/null
```
https://bbs.archlinux.org/viewtopic.php?pid=2162534#p2162534

**Comment by AladW** [comment]
Fixed in pacman 7.0.0: https://gitlab.archlinux.org/pacman/pacman/-/releases/v7.0.0
